includes/blocks/course-theme/class-course-navigation.php

```diff
@@ -189,20 +188,20 @@ function( $lesson ) {
 		}
 
 		return '
-			<section ' . \Sensei_Block_Helpers::render_style_attributes( $classes, [] ) . '>
-				<header class="sensei-lms-course-navigation-module__header">
+			<li ' . \Sensei_Block_Helpers::render_style_attributes( $classes, [] ) . '>
+				<div class="sensei-lms-course-navigation-module__header">
 					<button type="button" class="sensei-collapsible__toggle sensei-lms-course-navigation-module__button ' . $collapsed . '">
-						<h2 class="sensei-lms-course-navigation-module__title">' . $title . '</h2>
+						<div class="sensei-lms-course-navigation-module__title">' . $title . '</div>
```

Comment:
One last improvement I noticed that would be interesting, is identifying if it's expanded or not, otherwise, they wouldn't know if what comes next is part of that module or not (if it's expanded or not).

Maybe we could add 2 extra information:
- A prefix "Module:"
- A `span` with the text "Expanded" or "Collapsed"

Since we're using `sensei-collapsible__toggle` here, I imagine we could expand it through some data attribute that we use to show as `aria-label` if it's collapsed or expanded. Or we could also use some `span` as screen reader only based on the `collapsed` class. WDYT?

Reply:
Yup good idea. I'll mark it up with `aria-controls` and `aria-expanded`.

Reply:
Ah, awesome! TIL about that arias.
Just about the `aria-controls` we need to take care because the user could include multiple Course Navigation in their page. It also makes me think that maybe we could add `"multiple": false` to some of the blocks.

Reply:
Added in [481dcaa](https://github.com/Automattic/sensei/pull/4749/commits/481dcaa33bd6708916a3f6b131e1a813fa4f25aa)